tutorials/intro/plot_introduction.py

```diff
@@ -1,404 +1,332 @@
-# Define the inverse parameters:
-
-snr = 3.0
-lambda2 = 1.0 / snr ** 2
-method = "dSPM"
-
-##############################################################################
-# Compute the inverse solution:
-
-stc = apply_inverse(evoked, inverse_operator, lambda2, method)
+#     Both :class:`~mne.io.Raw` and :class:`~mne.Epochs` objects have
+#     :meth:`~mne.Epochs.get_data` methods that return the underlying data
+#     as a :class:`NumPy array <numpy.ndarray>`. Both methods have a ``picks``
+#     parameter for subselecting which channel(s) to return; ``raw.get_data()``
+#     has additional parameters for restricting the time domain. The resulting
+#     matrices have dimension ``(n_channels, n_times)`` for
+#     :class:`~mne.io.Raw` and ``(n_epochs, n_channels, n_times)`` for
+#     :class:`~mne.Epochs`.
+#
+# Next we'll pool across left/right stimulus presentations so we can compare
+# auditory versus visual responses. To avoid biasing our signals to the
+# left or right, we'll use :meth:`~mne.Epochs.equalize_event_counts` first to
+# randomly sample epochs from each condition to match the number of epochs
+# present in the condition with the fewest good epochs.
+
+conds_we_care_about = ['auditory/left', 'auditory/right',
+                       'visual/left', 'visual/right']
+epochs.equalize_event_counts(conds_we_care_about)  # this operates in-place
+aud_epochs = epochs['auditory']
+vis_epochs = epochs['visual']
+del raw, epochs  # free up memory
+
+###############################################################################
+# Like :class:`~mne.io.Raw` objects, :class:`~mne.Epochs` objects also have a
+# number of built-in plotting methods. One is :meth:`~mne.Epochs.plot_image`,
+# which shows each epoch as one row of an image map, with color representing
+# signal magnitude; the average evoked response and the sensor location are
+# shown below the image:
+
+aud_epochs.plot_image(picks=['MEG 1332', 'EEG 021'])
+
+###############################################################################
+# Estimating evoked responses
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# Now that we have our conditions in ``aud_epochs`` and ``vis_epochs``, we can
+# get an estimate of evoked responses to auditory versus visual stimuli by
+# averaging together the epochs in each condition. This is as simple as calling
+# the :meth:`~mne.Epochs.average` method on the :class:`~mne.Epochs` object,
+# and then using a function from the :mod:`mne.viz` module to compare the
+# global field power for each sensor type of the two :class:`~mne.Evoked`
+# objects:
 
-##############################################################################
-# Save the source time courses to disk:
+aud_evoked = aud_epochs.average()
+vis_evoked = vis_epochs.average()
 
-stc.save('mne_dSPM_inverse')
+mne.viz.plot_compare_evokeds(dict(auditory=aud_evoked, visual=vis_evoked),
+                             show_legend='upper left',
+                             show_sensors='upper right')
```

Comment:
the EEG channel layout in this plot is broken.

Reply:
also I find the fonts for some reason really small.

Reply:
I've noticed the EEG channel layouts being weird in some other plots too.  I'll open a separate issue.

Reply:
see #6304